mac/Keyman4MacIM/Keyman4MacIM/Logging/KMSentryHelper.m

```diff
@@ -0,0 +1,98 @@
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ *
+ * Created by Shawn Schantz on 2025-01-13.
+ *
+ * Description
+ */
+
+#import "KMSentryHelper.h"
+@import Sentry;
+
+@implementation KMSentryHelper
+
+NSString * const kApiComplianceTagName = @"apiCompliance";
+NSString * const kArchitectureTagName = @"architecture";
+NSString * const kOskVisibleTagName = @"oskVisible";
+NSString * const kClientAppIdTagName = @"clientAppId";
+NSString * const kKeyboardTagName = @"keyboard";
+NSString * const kHasAccessibilityTagName = @"accessibilityEnabled";
+NSString * const kActiveKeyboardCountTagName = @"activeKeyboardCount";
+NSString * const kLifecycleStateTagName = @"lifecycleState";
```

Comment:
Do these get sent to Sentry?

Reply:
yes, all are sent to Sentry with the `setTagValue` API